File: Person.py
class Person:

    def __init__(self, nombres:str, apellidos:str, edad:int, peso:float):
        self._nombres = nombres
        self._apellidos = apellidos
        self._edad = edad
        self._peso = peso

    def __str__(self) -> str:
        return f"{self._nombres} {self._apellidos} : {self._edad}, {self._peso:.2f}"

File: Person.java
class Person implements Comparable<Person> {

    private String nombres;
    private String apellidos;

    public Person(String n, String a) {
        nombres = n;
        apellidos = a;
    }

    public String toString() {
        return nombres + " " + apellidos;
    }

    @Override
    public int compareTo(Person o) {
        return 0;
    }

}